src/spec-node/devContainersSpecCLI.ts

```diff
@@ -696,17 +716,23 @@ function runUserCommandsOptions(y: Argv) {
 	})
 		.check(argv => {
 			const idLabels = (argv['id-label'] && (Array.isArray(argv['id-label']) ? argv['id-label'] : [argv['id-label']])) as string[] | undefined;
+			const isWorkspaceFound = argv['workspace-folder'] || findWorkspaceFolder();
+			console.debug(isWorkspaceFound);
```

Comment:
nit: Should we remove this?

Reply:
Do you mean the debug log? I just removed it. 
The `isWorkspaceFound` is required in a lower check, but I adjusted the Error Message there as well.